bundle/bundle.go

```diff
@@ -43,10 +43,26 @@ type Bundle struct {
 	AutoApprove bool
 }
 
+const ExtraIncludePathsKey string = "DATABRICKS_BUNDLE_INCLUDES"
+
 func Load(path string) (*Bundle, error) {
```

Comment:
If I understand correctly the behaviour is different from what is intended based on PR description

This will initialise it not only when BUNDLE_ROOT is set but also if there any bundle/databricks.yml in current or child dirs (https://github.com/databricks/cli/blob/main/bundle/root.go#L51-L56C9). 

Is this correct?

Reply:
Can we be more explicit here? Our condition to have an empty config is:
1. We have BUNDLE_ROOT env variable
2. There is no config file in this BUNDLE_ROOT

Could you codify this condition? It would be somethin like

```go
_, hasBundleRootEnv := os.LookupEnv(envBundleRoot)
configFile, noConfigFoundErr := config.FileNames.FindInPath(path)
if hasBundleRootEnv && noConfigFoundErr != nil {
  ...
} 
```